packages/devtools_app/lib/src/service_manager.dart

```diff
@@ -288,6 +288,24 @@ class ServiceConnectionManager {
 
     return fps.roundToDouble();
   }
+
+  bool libraryUriAvailableNow(String uri) {
+    assert(serviceAvailable.isCompleted);
+    assert(isolateManager.selectedIsolateAvailable.isCompleted);
+    return isolateManager.selectedIsolateLibraries
+        .map((ref) => ref.uri)
```

Comment:
duplicated code between libraryUriAvailableNow and libraryUriAvailable.  
`libraryUriAvailableNow` can be called after awaiting the `selectedIsolateAvailable.future`
the code can also be simplified to 
isolateManager.selectedIsolateLibraries.any((u) => u.uri.startsWith(uri))